modules/imagetools/src/main/ts/ephox/imagetools/util/Conversions.ts

```diff
@@ -1,4 +1,5 @@
 import { Optional, Type } from '@ephox/katamari';
+import { Ready, SugarElement } from '@ephox/sugar';
```

Comment:
Not something you've done here, but I did notice that katamari and sugar aren't listed as dependencies on the imagetools package 🤦  we should probably fix that when we move it out of the monorepo.

Reply:
So... about the things that would be fixed if we switched to Yarn Berry 😂